public/app/features/datasources/components/picker/DataSourceDropdown.tsx

```diff
@@ -359,3 +350,84 @@ function getStylesPickerContent(theme: GrafanaTheme2) {
     `,
   };
 }
+
+export interface FooterProps extends PickerContentProps {}
+
+function Footer({ onClose, onChange, onClickAddCSV, ...props }: FooterProps) {
+  const styles = useStyles2(getStylesFooter);
+  const isUploadFileEnabled = props.uploadFile && config.featureToggles.editPanelCSVDragAndDrop;
+
+  const onKeyDownLastButton = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
Even though we tried to avoid that handling of "onkeydown" manually for each button, this is a clean way to do it since we have to right now.